### DISCUSSION TITLE
Access to Gannet

### DISCUSSION THREAD

**Discussion by aspencoyle** [sr320/course-fish546-2021#97] [Q&A] [answered]
I'm having a hard time logging into Gannet from the command line. I've persistently had problems with Gannet - it just doesn't seem to like me! For instance, my password (which is definitely correct and works just fine when accessing via the website) is rejected when I try to log in via the command line.

Attempt to access Gannet: `ssh -v afcoyle@gannet.fish.washington.edu`

Response after entering password: `Permission denied, please try again.`

Part of the issue might be that I may have multiple accounts - when I log in online, it gives me multiple options - one with the username "afcoyle" and the other with "afcoyle@uw.edu". When I checked my password manager, the two have separate passwords.

I have absolutely nothing on Gannet (largely due to these issues), so if the easiest route is deleting all accounts associated with me and starting from scratch, that's fine by me!

**Comment by kubu4** [comment]
Please post your IP address (IP4; should be formatted something like ###.###.###.##).

**Comment by kubu4** [comment, marked as answer]
Alrighty, I checked your IP address and it wasn't blocked. So, I reset your password. Check Slack DM for info.

**Reply by aspencoyle** [reply]
Woohoo, it worked! Thanks, Sam!
